vulnerabilities/importers/github.py

```diff
@@ -146,189 +122,339 @@
     "-class.-jw.util.version.Version-",
 }
 
+ECOSYSTEM_PKG_TYPE_MAPPING = {
```

Comment:
What about this more explicit name?
```suggestion
PACKAGE_TYPE_BY_GITHUB_ECOSYSTEM = {
```